sayn/core/config.py

```diff
@@ -52,22 +52,33 @@ class Project(BaseModel):
     default_db: Optional[str]
     parameters: Optional[Dict[str, Any]] = dict()
     presets: Optional[Dict[str, Dict[str, Any]]] = dict()
-    dags: List[str]
+    task_groups: List[str] = []
 
     @validator("required_credentials")
     def required_credentials_are_unique(cls, v):
         return is_unique("required_credentials", v)
 
-    @validator("dags")
-    def dags_are_unique(cls, v):
-        return is_unique("dags", v)
-
     @validator("default_db")
     def default_db_exists(cls, v, values, **kwargs):
         if v not in values["required_credentials"]:
             raise ValueError(f'default_db value "{v}" not in required_credentials')
         return v
 
+    @validator("task_groups", pre=True, always=True)
+    def set_task_groups(cls, v):
+        for file_name in os.listdir(Path("tasks")):
```

Comment:
Alternative code: `groups = [f.name[:-5] for f in Path('tasks').glob('*.yaml')]`